web/packages/core/src/public-api.ts

```diff
@@ -72,10 +72,10 @@ export class PublicAPI {
                 prev.superseded();
             } else if (
                 prev.constructor === Object &&
-                prev.config instanceof Object
+                prev["config"] instanceof Object
             ) {
                 /// We're the first, install user configuration
-                this.config = prev.config;
+                this.config = prev["config"];
```

Comment:
let's be consistent and call it `this["config"]`

Reply:
These are different - `this.config` is a class field while `prev` is an object defined on `window` (i.e. `window.RufflePlayer.config = { ... }`). Due to [noPropertyAccessFromIndexSignature](https://www.typescriptlang.org/tsconfig#noPropertyAccessFromIndexSignature), the latter must be `prev["config"]`, but I think we should write `this.config` where allowed.